pkg/function/function.go

```diff
@@ -193,7 +193,12 @@ func (f Function) Actions(ctx context.Context) ([]inngest.ActionVersion, []innge
 }
 
 func (f Function) action(ctx context.Context, s Step, n int) (inngest.ActionVersion, error) {
-	id := fmt.Sprintf("%s-step-%d", f.ID, n)
+	suffix := "-test"
+	if state.IsProd() {
+		suffix = "-prod"
+	}
+
+	id := fmt.Sprintf("%s-step-%d-%s", f.ID, n, suffix)
```

Comment:
This looks like the suffix already has `-` so this would be `<id>-step-2--prod` ?

Reply:
Thanks for catching!  Fixed.